lib/subproviders/geth_api_double.js

```diff
@@ -519,4 +519,301 @@ GethApiDouble.prototype.debug_traceTransaction = function(tx_hash, params, callb
   this.state.queueTransactionTrace(tx_hash, params, callback);
 };
 
+/* 
+  RPC AUDIT:  
+  TODO ETH: eth_getUncleCountByBlockHash, eth_getUncleCountByBlockNumber, eth_getUncleByBlockHashAndIndex,
+        eth_getUncleByBlockNumberAndIndex, eth_newPendingTransactionFilter,
+        eth_getFilterLogs, eth_getWork, eth_submitWork, eth_submitHashrate
```

Comment:
eth_getFilterLogs is implemented by the `FilterSubprovider` in the `web3-provider-engine` module.

Reply:
Fixed this one myself